core-java-modules/core-java-concurrency-advanced-4/src/main/java/com/baeldung/ownablesynchronizers/Application.java

```diff
@@ -0,0 +1,66 @@
+package com.baeldung.ownablesynchronizers;
+
+import java.util.concurrent.locks.ReentrantLock;
+
+public class Application {
+
+    static ReentrantLock firstLock = new ReentrantLock(true);
+    static ReentrantLock secondLock = new ReentrantLock(true);
+
+    public static void main(String[] args) throws Exception {
+        // Create first thread that will lock on 'firstLock'
```

Comment:
This and the below comment about when the thread will be locked is not adding much value. Could you remove it or modify the code to express better your purpose??